doc/source/working_with_kiwi/xml_description.rst

```diff
@@ -192,8 +192,8 @@ sections):
 - `vmx`: build a virtual machine image, see: :ref:`vmx`
 - `oem`: results in an expandable image that can be deployed via a bootable
   installation medium, e.g. a USB drive or a CD. See :ref:`oem`
-- `pxe`: creates an image that can be booted via PXE (network boot), see
-  :ref:`build_pxe`
+- `pxe`: creates an image that can be booted from the network using the
+  legacy kiwi netboot infrastructure, see :ref:`build_legacy_pxe`
```

Comment:
Shouldn't we include the `kis`type here?

Reply:
done